drivers/osdp/src/osdp_common.h

```diff
@@ -0,0 +1,404 @@
+/*
+ * Copyright (c) 2020 Siddharth Chandrasekaran <siddharth@embedjournal.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _OSDP_COMMON_H_
+#define _OSDP_COMMON_H_
+
+#include <drivers/osdp.h>
+#include <assert.h>
+
+#ifndef NULL
+#define NULL                   ((void *)0)
+#endif
+#define TRUE                   (1)
+#define FALSE                  (0)
```

Comment:
You do not need these, just remove them.
Use `true / false` instead of `TRUE / FALSE` in the code.

Reply:
Done.